app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/SuggestionBottomBar.kt

```diff
@@ -27,7 +27,7 @@ import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 
 @Composable
-fun SuggestionBottomBar(onSuggestionClick: () -> Unit) {
+fun SuggestionBottomBar(onSuggestionClick: () -> Unit = {}) {

```

Comment:
I think that this is a good practice for this type of composable

Reply:
Thank you!